ethereum/contracts/ETHApp.sol

```diff
@@ -14,7 +14,7 @@ contract ETHApp is Application {
     uint256 public nonce;
     uint256 public totalETH;
 
-    event Transfer(address _sender, bytes32 _recipient, uint256 _amount);
+    event AppTransfer(address _sender, bytes32 _recipient, uint256 _amount);
```

Comment:
Ooh, we can't make this change without updating both the Relayer and Parachain. Is there a compelling reason why we should rename?

Reply:
@vgeddes See above comment

Reply:
Oh I see! Sorry I must have missed the other comment. @denalimarsh I'm gonna push the necessary parachain and relayer changes to this branch too.